lib/public/IConfig.php

```diff
@@ -257,4 +257,16 @@ public function deleteAppFromAllUsers($appName);
 	 * @since 8.0.0
 	 */
 	public function getUsersForUserValue($appName, $key, $value);
+
+
+	/**
+	 * Clear all the cached app config values
+	 *
+	 * WARNING: do not use this - this is only for usage with the SCSSCacher to
+	 * clear the memory cache of the app config
+	 *
+	 * @since 21.0.0
```

Comment:
```suggestion
	 * @internal 
```

might be more suitable